be/src/storage/tablet_manager.cpp

```diff
@@ -840,7 +840,7 @@ Status TabletManager::load_tablet_from_dir(DataDir* store, TTabletId tablet_id,
     int32_t shard = stol(shard_str);
     // load dir is called by clone, restore, storage migration
     // should change tablet uid when tablet object changed
-    if (Status st = TabletMeta::reset_tablet_uid(header_path); !st.ok()) {
+    if (!TabletMeta::reset_tablet_uid(header_path).ok()) {
```

Comment:
header_path can use meat_path instead.
header is a old name which is already non-sense.